src/core/include/openvino/op/util/evaluate_helpers.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/partial_shape.hpp"
+#include "openvino/runtime/tensor.hpp"
+
+namespace ov {
+namespace op {
+namespace util {
+
+/**
+ * @brief Get the tensors shapes as ov::PartialSHape
+ *
+ * @param tensors  Input tensors vector to get its shapes.
+ * @return Vector of partial shapes sam size as input tensor vector.
+ */
+std::vector<PartialShape> get_tensors_shapes(const TensorVector& tensors);
```

Comment:
Why not `ov::Shape`?

Reply:
As it easier to use with `shape_infer` functions.
Will rename it to `get_tensors_partial_shapes` to be more explicit what this function will return.